internal/cache/metadata/test/stat_cache_test.go

```diff
@@ -12,13 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package caching_test
+package test
```

Comment:
Why are moving this inside test package?

Generally we keep the go file and their unit test on the same level.

Reply:
@raj-prince  This looks like old code. I'll see what went wrong. Thanks for pointing out..

Reply:
Found the issue. Pushed the updated code, which uses package metadata_test, as it should.  Marking this resolved.